src/main/java/com/solvd/RentalCompany/exceptionHandlers/IObjectNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.solvd.RentalCompany.exceptionHandlers;
```

Comment:
I have one class with id of String type (Category class). In fact, I made this interface only because of him, because in the rest of the classes id is of type Integer. The exception handler has a method to get the id of the object that the exception was thrown on, and this interface allows me to determine the type of that id